src/ui/browserAction/popup.html

```diff
@@ -34,7 +34,16 @@
       <open-mozilla-vpn-message-screen
         slot="MessageOpenMozillaVPN"
       ></open-mozilla-vpn-message-screen>
-      <!-- TODO: Split Tunnel Message, Onboarding, Update Message -->
+      <onboarding-screen-1
```

Comment:
With this approach i can see why the pagination would be hard to add (agreed) 

WDYT: 
one new `Screen` Element -> "Onboarding" that is shown here if onboarding is not complete. 

The Onboarding screen then would be something like 
```html
<main>
<conditional-view slotName=${this.onboardingCount} >
  <slot=1 onboarding-screen-1>
  <slot=2 onboarding-screen-2>
  <slot=3 onboarding-screen-3>
  <slot=4 onboarding-screen-4>
</conditional-view>
<pagination>
<button-next>

</main>


```

Reply:
(this is also a nonblocking suggestion, if we REALLY dont want the indicators let's not add this )

Reply:
I'm not fixated on the pagination either (MAC doesn't have them) but I do think they're a very nice touch, if not too painful.

Reply:
Pagination is now in here - will update the video once I add in the new fourth screen.